src/pages/en/guides/typescript.md

```diff
@@ -28,14 +28,30 @@ Some TypeScript configuration options require special attention in Astro. Below
     "resolveJsonModule": true,
     // Enable stricter transpilation for better output.
     "isolatedModules": true,
-    // Add type definitions for our Astro runtime.
-    "types": ["astro/client"],
-    // Tell TypeScript where your build output is
-    "outDir": "./dist"
+    // Astro will directly run your TypeScript code, no transpilation needed.
+    "noEmit": true
   }
 }
 ```
 
+Additionally, our templates include `env.d.ts` file inside the `src` folder that contains the following:
+
+```typescript title="env.d.ts"
+/// <reference types="astro/client" />
+```
+
+This will includes [Vite's client types](https://vitejs.dev/guide/features.html#client-types) inside your project. Alternatively, if you prefer not having this file in your project, you can delete it and use the [`types` setting](https://www.typescriptlang.org/tsconfig#types) inside your `tsconfig.json` to add them:
+
+```json title="tsconfig.json"
+{
+  "compilerOptions": {
+    "types": ["astro/client"]
+  }
+}
+```
```

Comment:
```suggestion
Additionally, our templates include an `env.d.ts` file inside the `src` folder to provide [Vite's client types](https://vitejs.dev/guide/features.html#client-types) to your project:

```typescript title="env.d.ts"
/// <reference types="astro/client" />
```
Optionally, you can delete this file and instead add the [`types` setting](https://www.typescriptlang.org/tsconfig#types) to your `tsconfig.json`:

```json title="tsconfig.json"
{
  "compilerOptions": {
    "types": ["astro/client"]
  }
}
```
```

Reply:
Hi @Princesseuh! Just a reminder that this is still here, waiting for you when you're ready!